proto/frequenz/api/common/v1/grid.proto

```diff
@@ -0,0 +1,62 @@
+// Frequenz definitions of grids as entites participating in trading.
+//
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package frequenz.api.common.v1.grid;
```

Comment:
> Rename this package, e.g. to trade_delivery or something similar, OR

Well these things are given/provided by the grid operator and not necessary just related to trading. The assets API is also not really about trading at all. These things are really related to the grid. They only apply if a microgrid is connected to a grid. I don't have a better idea right now tbh.

> Place the delivery related messages in microgrid.proto, because they seem to be tighly related to the microgrid domain

I'm not sure. `Counterparty` of example has nothing to do with the microgrid.

> Rename the package components.grid to components.grid_connection_point.

This feels misleading. These things are like the config parameters for the grid that a microgrid is connected to. Its like the metadata of the grid and how its being operated.

Reply:
Also if you look at this import path that looks totally legit to me. I mean its the delivery area of the grid, isn't it?

> frequenz.api.common.v1.grid.DeliveryArea

Reply:
Okay, so we leave it as is?

Reply:
Well I don't have a better idea right now.

Reply:
Okay, then we can keep it as for now. We can revisit later if needed.